src/views/pages/PageDetail.vue

```diff
@@ -71,7 +71,7 @@ import { EventBus } from '../../eventBus.js'
 				</div>
 				<div>
 					<b>Laatst bijgewerkt:</b>
-					<span>{{ pageStore.pageItem.updatedAt }}</span>
+					<span>{{ pageStore.pageItem?.updated_at?.toLocaleDateString() || "-" }}</span>
```

Comment:
This still should be updatedAt, but I think something in the backend changed. 
I will make an issue for this